docker-compose/docker-compose.yml

```diff
@@ -124,3 +124,15 @@ services:
     - ./volumes/worker-hardlinking-ubuntu22-04:/worker
     depends_on:
     - runner-installer
+
+  remote-asset:
+    # TODO(minor-fixes): Replace with actual container once pushed to ghcr
+    image: bazel/cmd/bb_remote_asset:bb_remote_asset_container
```

Comment:
This can't be merged as-is until this image is put somewhere public (ghcr.io?)

Reply:
Would you mind updating the image reference now when `ghcr.io/buildbarn/bb-remote-asset:20230910T205857Z-52b25db` is available?